spec/wrappers/wakes/google_analytics_api_wrapper_spec.rb

```diff
@@ -0,0 +1,137 @@
+require 'rails_helper'
+
+RSpec.describe Wakes::GoogleAnalyticsApiWrapper do
+  describe '#get_pageviews_for_path' do
+    # Describes this view: https://www.google.com/analytics/web/?hl=en#report/content-pages/a1853263w3269662p8231065/%3F_u.date00%3D20140101%26_u.date01%3D20141231%26explorer-table.plotKeys%3D%5B%5D%26explorer-table.advFilter%3D%5B%5B0%2C%22analytics.pagePath%22%2C%22RE%22%2C%22%5E%2Fabout(%5C%5C%3F%7C%24)%22%2C0%5D%5D%26explorer-table.rowCount%3D100/
+    it 'returns the total pageviews for the given date range' do
+      pageviews = subject.get_pageviews_for_path('/about', :start_date => '2014-01-01', :end_date => '2014-12-31')
+
+      expect(pageviews).to eq 78_891
+    end
+
+    describe 'handles lang params' do
+      it 'only matches if original path has lang' do
+        pageviews = subject.get_pageviews_for_path('/sermons/for-his-sake-and-for-your-joy-go-low?lang=es',
```

Comment:
@benhutton I know this is just copied from the old spec, but are we doing live api lookups in these specs? Or is VCR implicit here?